string dimension
float64 value
lvr_ros/Mesh mesh
float64 grid_resolution
float64 grid_unoccupied_default
std_msgs/ColorRGBA color
---
nav_msgs/OccupancyGrid grid
visualization_msgs/MarkerArray marker
